packages/plugin-user/src/user.js

```diff
@@ -274,15 +274,21 @@ const User = SparkPlugin.extend({
         })
         .catch((err) => Promise.reject(new Error(`failed to set authorization`, err)))
       )
-      .then(() => this.request({
-        service: `atlas`,
-        resource: `users/activations`,
-        method: `POST`,
-        headers,
-        body: options,
-        shouldRefreshAccessToken,
-        requiresClientCredentials
-      }))
+      .then(() => {
+        if (typeof window === `undefined`) {
+          headers[`user-agent`] = `spark-js-sdk`;
+        }
+
```

Comment:
No need to check headers[`user-agent`] since headers is initialized right above.